v21.1/stream-data-out-of-cockroachdb-using-changefeeds.md

```diff
@@ -308,6 +308,10 @@ In this example, you'll set up a changefeed for a single-node cluster that is co
 
 3. Download and extract the [Confluent Open Source platform](https://www.confluent.io/download/) (which includes Kafka).
 
+{{site.data.alerts.callout_info}}
+Currently, [changefeeds](stream-data-out-of-cockroachdb-using-changefeeds.html) connected to [Kafka versions < v1.0](https://docs.confluent.io/platform/current/installation/versions-interoperability.html) are not supported in CockroachDB v21.1.
+{{site.data.alerts.end}}
```

Comment:
I leave it to you, but I think we could probably avoid this callout here.  If the user is downloading confluent.io for the first time via these instructions, they are unlikely to be downloading an End-of-Life version.

Reply:
Yes, that's a v. good point and I agree. I probably went slightly overboard with the callouts :-) I think the other sections would suffice... Thanks for the comment!